srcpkgs/quicklisp/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'quicklisp'
+pkgname=quicklisp
+version=20150128
+revision=1
+wrksrc=${pkgname}
+build_style=fetch
+makedepends="sbcl"
+depends="sbcl"
+short_desc="Library manager for Common Lisp"
+maintainer="Rahul <basicfunc@gmail.com>"
+license="custom:bsd"
+homepage="https://beta.quicklisp.org"
+distfiles="${homepage}/quicklisp.lisp"
+checksum=4a7a5c2aebe0716417047854267397e24a44d0cce096127411e9ce9ccfeb2c17
+nocheckperms=yes
+
+do_install( ){
+    echo -ne "(quicklisp-quickstart:install :path \"${DESTDIR}/usr/lib/${pkgname}\")\n(quit)\n" | sbcl --no-userinit --load $pkgname.lisp
+    chmod 777 -R "$DESTDIR"/usr/lib/${pkgname}
+    chmod 777 -R "$DESTDIR"/usr/lib/${pkgname}/tmp
+    chmod 777 -R "$DESTDIR"/usr/lib/${pkgname}/dists
+    chmod 777 -R "$DESTDIR"/usr/lib/${pkgname}/local-projects
```

Comment:
This is very weird, why?

Reply:
Oh that's what I tried to fix the same error and forgot to fix.

Reply:
Fixed